lib/util/esWrapper.js

```diff
@@ -152,9 +152,13 @@ class ESWrapper {
       const matches = message.match(mapping.regex);
 
       if (matches) {
-        return errorsManager.get(
+        console.log(mapping.subcode)
+        console.log(mapping.getPlaceholders(matches))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)